docs/overview_and_guide.rst

```diff
@@ -69,7 +70,8 @@ Installation from source
 
 Clone the repository::
 
-    git clone https://github.com/raiden-network/raiden.git
+    git clone https://github.com/raiden-network/raiden.git |checkout_branch|
+    git checkout |version|
```

Comment:
If version is `nightly` then this would become
> `git checkout nightly`

which fails. Is that even necessary with the branch selection in the clone command?

Reply:
probably it's just a leftover. Guess the checkout is not needed there.